tests/alembic/revisions/test_d429de0e5338_adopt_peer_paradigm.py

```
@@ -0,0 +1,286 @@
+
+@register_before_upgrade("d429de0e5338")
+def prepare_peer_paradigm(verifier: MigrationVerifier) -> None:
+    """Seed legacy tables so the migration can transform real data."""
+
+    schema = verifier.schema
+    conn = verifier.conn
+
+    conn.execute(
+        text(
+            f'INSERT INTO "{schema}"."apps" ("public_id", "name") '
+            + "VALUES (:app_id, :app_name)"
```

Ruff (0.14.1)

Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)